From 1932843a310e50754cd4cdb9731f03f19c20e81c Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Mon, 29 Jan 2024 20:51:12 -0800
Subject: [PATCH 3/6] add lib64 to libhwloc search path

cmake on ubuntu23.10 finds library only from
lib, not from lib64w

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 openmp/runtime/cmake/config-ix.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/openmp/runtime/cmake/config-ix.cmake b/openmp/runtime/cmake/config-ix.cmake
index 23b56d5f06b4..23b1a2df7931 100644
--- a/openmp/runtime/cmake/config-ix.cmake
+++ b/openmp/runtime/cmake/config-ix.cmake
@@ -352,7 +352,8 @@ if(${LIBOMP_USE_HWLOC})
   set(CMAKE_REQUIRED_INCLUDES)
   find_library(LIBOMP_HWLOC_LIBRARY
     NAMES hwloc libhwloc
-    HINTS ${LIBOMP_HWLOC_INSTALL_DIR}/lib)
+    HINTS ${LIBOMP_HWLOC_INSTALL_DIR}/lib
+          ${LIBOMP_HWLOC_INSTALL_DIR}/lib64)
   if(LIBOMP_HWLOC_LIBRARY)
     # In case libhwloc is static, check_library_exists does not work on static libs
     get_filename_component(LIBOMP_HWLOC_LIBRARY_EXT ${LIBOMP_HWLOC_LIBRARY} EXT)
-- 
2.43.0

